runtime/startup_byt.c

```diff
@@ -404,7 +404,7 @@ static void do_print_config(void)
          "false");
 #endif
   printf("windows_unicode: %s\n",
-#if WINDOWS_UNICODE
+#if defined(WINDOWS_UNICODE) && WINDOWS_UNICODE
```

Comment:
I'd prefer
```suggestion
#if defined(WINDOWS_UNICODE) && WINDOWS_UNICODE != 0
```
for readability and clarity, then.

Reply:
Thanks, applied.